staticjinja/staticjinja.py

```diff
@@ -374,7 +379,11 @@ def make_site(searchpath="templates",
 
     loader = FileSystemLoader(searchpath=searchpath,
                               encoding=encoding)
-    environment = Environment(loader=loader, extensions=extensions or [])
+    env_kwargs.pop("loader", None)
+    env_kwargs.pop("extensions", None)
+    env_kwargs.pop("filters", None)
```

Comment:
Actually, rather than `pop`ing these, I think it makes more sense to set their defaults to `extensions` and `filters`, excluding `loader` of course. It seems like `env_kwargs` should be the preferred way to pass values to the environment, since it's more flexible. So I'd write it like this:

```
del env_kwards['loader']
env_kwargs.setdefault('extensions', extensions)
env_kwargs.setdefault('filters', filters)
```

And then remove the code setting filters, and the `extensions` keyword argument to `Environment`.